llvm/lib/SYCLLowerIR/ESIMD/ESIMDVerifier.cpp

```diff
@@ -34,8 +34,21 @@ namespace id = itanium_demangle;
 static const char *LegalSYCLFunctions[] = {
     "^sycl::_V1::accessor<.+>::accessor",
     "^sycl::_V1::accessor<.+>::~accessor",
-    "^sycl::_V1::accessor<.+>::getNativeImageObj",
     "^sycl::_V1::accessor<.+>::__init_esimd",
+    "^sycl::_V1::accessor<.+>::get_pointer.+",
+    "^sycl::_V1::accessor<.+>::getPointerAdjusted",
+    "^sycl::_V1::accessor<.+>::getQualifiedPtr",
+    "^sycl::_V1::accessor<.+>::getTotalOffset",
+    "^sycl::_V1::accessor<.+>::getLinearIndex",
```

Comment:
Previously, we only allowed accessor functions in stateless mode. Now it looks like we are allowing them even in stateful mode. I'd understand if we started alloing local accessors only, but it seems we are allowing them for normal accessors.  Can you help me understand this?

Reply:
I added such a comment to the description after your question:

Previously ESIMD did not let working with accessors because GPU VC BE required using
only special GenX intrinsics for loads/stores and did not support regular LLVM IR loads/stores.
Now it does.

Reply:
ah ok thx